modules/troubleshoot/partials/debug-bundle.adoc

```diff
@@ -168,550 +170,260 @@ For a description of the files and directories, see <<Contents of the debug bund
 
 See xref:console:ui/generate-bundle.adoc[].
 
-// tag::inspect[]
-== Inspect the debug bundle
-
-After downloading the debug bundle files, you can inspect the contents to debug your cluster. This section provides some useful data points to check while troubleshooting.
-
-Most files in the debug bundle are JSON files. To make it easier to read these files, this section uses jq. To install jq, see the https://stedolan.github.io/jq/download/[jq downloads page^].
+== Automatically remove debug bundles
 
-=== View the version of Redpanda on all brokers
+To prevent debug bundles from accumulating in the storage directory, you can configure the xref:reference:properties/cluster-properties.adoc#debug_bundle_auto_removal_seconds[`debug_bundle_auto_removal_seconds`] property. This cluster configuration property automatically deletes debug bundles after the specified number of seconds. By default, this property is not set, meaning debug bundles are retained indefinitely.
```

Comment:
```suggestion
To avoid manually deleting debug bundles, you can configure the xref:reference:properties/cluster-properties.adoc#debug_bundle_auto_removal_seconds[`debug_bundle_auto_removal_seconds`] property. This cluster configuration property automatically deletes debug bundles after the specified number of seconds. By default, this property is not set, meaning debug bundles are retained indefinitely.
```